### ISSUE TITLE
“gnparser” cannot be opened because the developer cannot be verified.

### ISSUE THREAD

**Issue by jar398** [gnames/gnparser#236]
the README says: "To use GNparser as a command line tool under Windows, Mac or Linux, download the [latest release](https://github.com/gnames/gnparser/releases/latest), uncompress it, and copy gnparser binary somewhere in your PATH."

So I did as directed, tried to run from the shell, and got this error message in a popup:

> “gnparser” cannot be opened because the developer cannot be verified.

Since MacOS isn't mentioned in the installation section of the README, my guess is that you can't support it on the Mac any more, so the mention of "Mac" at the top no longer applies.  My recommendation is to remove 'Mav' from the README.  But of course if there is a solution that would be great!

**Comment by dimus** [comment]
Thanks for letting me know @jar398, this is very strange, I will try to reproduce the problem on Windows. Mac is supported, and it is pretty much the same procedure as for Linux: https://github.com/gnames/gnparser#linux-or-os-x

Homebrew is enother way to install it on Mac: https://github.com/gnames/gnparser#install-with-homebrew

**Comment by dimus** [comment]
I added  'Mac' to docs with https://github.com/gnames/gnparser/commit/80acf2cb8ce46dd3779d4f487cbcabf69fd8c185 for clarity

**Comment by jar398** [comment]
Yeah, I was referring to the quickstart instructions at the top of the readme.

But I did find out how to get it to go.  In the security panel in system preferences select accept from other developers. Then try to open; it will fail. Then look in the security panel again and there will be a question asking whether you want to run gnparser in spite of it being unidentified developer.  Say yes.  Then try again.

I'm running 10.15.7.  Of course things will change in the future...

**Comment by jar398** [comment]
This is all about the gradually tightening MacOS security model. They seem to be moving in the direction of allowing only App Store applications even on laptops. I doubt you'll learn anything by checking behavior on Windows.

**Comment by dimus** [comment]
ah, I see, I understood now that it was a Mac 10.15.7 security problem. That is very unfortunate. I wonder if brew method is still working without  doing system preferences modification.

**Comment by dimus** [comment]
Does this sound right, @jar398?

 https://github.com/gnames/gnparser/commit/79c2dc180828e4c632a7160640a0ebd708dca2c8

**Comment by jar398** [comment]
Yes, that will do.  I'll see if brew installation still works in the next week, sorta busy right now.

**Comment by jar398** [comment]
I'm sorry but my mac is too old for the required Xcode update (because it in turn requires an operating system upgrade), so I won't be able to do this test.  I've been able to use brew to install other things, so I think it's a good gamble that it will also work for gnparse. I can ask a friend to test the installation via brew but no promises.

**Comment by dimus** [comment]
Looks like people do not have problem installing gn apps with brew. Closing this ticket.
